src/nmr_to_structure/prepare_input/prepare_nmr_rxn_input.py

```diff
@@ -337,7 +337,7 @@ def main(
     # Load dataframe containing nmr data
     logger.info("Reading data.")
     nmr_df = pd.read_pickle(nmr_data)
-    nmr_df.drop(columns=["1H_NMR_exp", "13C_NMR_exp"], inplace=True)
+    #nmr_df.drop(columns=["1H_NMR_exp", "13C_NMR_exp"], inplace=True)
```

Comment:
If not necessary anymore: delete it. Otherwise, you can specify why it is commented out as a comment on the line before